Android and Android TV Monitoring Formatting Edit (#966)
DataDog/dd-sdk-android: docs/rum_getting_started.md

{{< /tabs >}}
{{< /site-region >}}

The initialization credentials require your application's variant name and uses the value of `BuildConfig.FLAVOR`. If you do not have variants, the credentials uses an empty string. 

> This enables the appropriate ProGuard `mapping.txt` file to automatically upload at build time so you can view deobfuscated RUM error stack traces. For more information, see the [Track Android Errors][8].

Reviewer: Nope, the upload is performed by the Gradle plugin, not the SDK. 
Adding the variant actually allows us to match the errors reported from the app with the mapping files uploaded by the gradle plugin

Author: ```suggestion
The Gradle plugin automatically uploads the appropriate ProGuard `mapping.txt` file at build time so you can view deobfuscated RUM error stack traces. For more information, see the [Track Android Errors][8].
```
